plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -390,12 +403,21 @@ def _query_and_results(self, client, sql, conn, job_params, timeout=None):
 
     def _retry_and_handle(self, msg, conn, fn):
         """retry a function call within the context of exception_handler."""
+        def reopen_conn_on_error(error):
+            for type in REOPENABLE_ERRORS:
+                if isinstance(error, type):
```

Comment:
I think you can just do `if isinstance(error, REOPENABLE_ERRORS)` here, right?

Reply:
TIL. Done.